packages/fiori/src/ViewSettingsDialog.js

```diff
@@ -682,10 +682,13 @@ class ViewSettingsDialog extends UI5Element {
 	 * This method can be used after the dialog is initially open, as the dialog need to set its initial settings.
 	 * The <code>ui5-view-settings-dialog</code> throws an event called "before-open", this can be used as trigger point.
 	 * The object should have the following format:
-	 * <code>{
-	 *	{ "sortOrder" : "Ascending", "sortBy" : "Name", "filters" : [{"Filter 1": ["Some filter 1", "Some filter 2"]}, {"Filter 2": ["Some filter 4"]}]}
-	 * }</code>
-	 * @param {string} settings A value to be set as predefined settings.
+	 * <code>
+	 *  {sortOrder: "Ascending", sortBy: "Name", filters: [{Filter 1: ["Some filter 1", "Some filter 2"]}, {Filter 2: ["Some filter 4"]}]}
```

Comment:
Not sure this will work... Filter 1 is property of an object, but also it is string key, and it contains space, so it must be quoted, otherwise it throws an error!
{sortOrder: "Ascending", sortBy: "Name", filters: [{**"Filter 1"**: ["Some filter 1", "Some filter 2"]}, {**"Filter 2"**: ["Some filter 4"]}]}

Reply:
I've updated the PR based on your comment.

```
{sortOrder: "Ascending", sortBy: "Name", filters: [{"Filter 1": ["Some filter 1", "Some filter 2"]}, {"Filter 2": ["Some filter 4"]}]}
```